share/functions/fish_vi_key_bindings.fish

```diff
@@ -1,3 +1,77 @@
+alias vi_dec 'vi_inc_dec dec'
+alias vi_inc 'vi_inc_dec inc'
+
+# TODO: Currently we do not support hexadecimal and octal values.
+function vi_inc_dec --description 'increment or decrement a number'
+  # The cursor is zero based, but all string functions assume 1 to be
+  # the lowest index. Adjust accordingly.
+  set cursor (math (commandline --cursor) + 1)
+  set line (commandline --current-buffer)
```

Comment:
doesn't look like this works for multiline command lines.
We can probably fix this by adding `| string collect` inside every
command substitution that can produce multiline output.
AFAICT that's only this line and some of the "string sub" calls.